TraderJoe-Bots/Large-Swap/src/agent.spec.ts

```diff
@@ -0,0 +1,512 @@
+import { FindingType, FindingSeverity, Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+import { TestTransactionEvent, createAddress, MockEthersProvider, MockEthersSigner } from "forta-agent-tools/lib/tests";
+import { BigNumber } from "ethers";
+import { Interface, keccak256, getCreate2Address } from "ethers/lib/utils";
+import { provideHandleTransaction } from "./agent";
```

Comment:
```suggestion
import { provideHandleTransaction, provideInitialize } from "./agent";
```
Related to `agent.ts` refactor.